src/Neo/SmartContract/Native/TokenManagement.NonFungible.cs

```diff
@@ -0,0 +1,289 @@
+
+    /// <summary>
+    /// Returns an iterator over the unique ids of NFTs for the specified asset (collection).
+    /// The iterator yields the stored unique id keys (UInt160) indexed under the NFT asset id.
+    /// </summary>
+    /// <param name="snapshot">A readonly view of the storage.</param>
+    /// <param name="assetId">The asset (collection) identifier whose NFTs are requested.</param>
+    /// <returns>
+    /// An <see cref="IIterator"/> that enumerates the NFT unique ids belonging to the given collection.
+    /// The iterator is configured to return keys only and to remove the storage prefix.
+    /// </returns>
+    /// <exception cref="InvalidOperationException">Thrown when the specified asset id does not exist.</exception>
+    /// <remarks>
+    /// The returned iterator is backed by the storage layer and uses the NFT asset-to-unique-id index.
+    /// Consumers should dispose the iterator when finished if they hold unmanaged resources from it.
+    /// </remarks>
+    [ContractMethod(CpuFee = 1 << 22, RequiredCallFlags = CallFlags.ReadStates)]
+    public IIterator GetNFTs(IReadOnlyStore snapshot, UInt160 assetId)
+    {
+        StorageKey key = CreateStorageKey(Prefix_TokenState, assetId);
+        if (!snapshot.Contains(key))
+            throw new InvalidOperationException("The asset id does not exist.");
+        const FindOptions options = FindOptions.KeysOnly | FindOptions.RemovePrefix;
+        var prefixKey = CreateStorageKey(Prefix_NFTAssetIdUniqueIdIndex, assetId);
+        var enumerator = snapshot.Find(prefixKey).GetEnumerator();
+        return new StorageIterator(enumerator, 21, options);
+    }
+
+    /// <summary>
+    /// Returns an iterator over the unique ids of NFTs owned by the specified account.
+    /// The iterator yields the stored unique id keys (<see cref="UInt160"/>) indexed under the NFT owner index.
+    /// </summary>
+    /// <param name="snapshot">A readonly view of the storage.</param>
+    /// <param name="account">The account whose NFTs are requested.</param>
+    /// <returns>
+    /// An <see cref="IIterator"/> that enumerates the NFT unique ids owned by the given account.
+    /// The iterator is configured to return keys only and to remove the storage prefix.
+    /// </returns>
+    /// <remarks>
+    /// The returned iterator is backed by the storage layer and uses the NFT owner-to-unique-id index.
+    /// Consumers should dispose the iterator when finished if they hold unmanaged resources from it.
+    /// </remarks>
+    [ContractMethod(CpuFee = 1 << 22, RequiredCallFlags = CallFlags.ReadStates)]
+    public IIterator GetNFTsOfOwner(IReadOnlyStore snapshot, UInt160 account)
+    {
+        const FindOptions options = FindOptions.KeysOnly | FindOptions.RemovePrefix;
+        var prefixKey = CreateStorageKey(Prefix_NFTOwnerUniqueIdIndex, account);
+        var enumerator = snapshot.Find(prefixKey).GetEnumerator();
+        return new StorageIterator(enumerator, 21, options);
```

Comment:
```suggestion
        return new StorageIterator(enumerator, Prefix_NFTOwnerUniqueIdIndex, options);
```

Reply:
`21` is the prefix length to be removed.